### DISCUSSION TITLE
Welcome to upbge Discussions!

### DISCUSSION THREAD

**Discussion by lordloki** [UPBGE/upbge#1347] [Announcements & others]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * If you don't know if it is a bug. please open a discussion first.

**Comment by sijiayidejinshuyun** [comment]
hi
When the motion actuator sets the speed for a dynamic object, it sometimes fails:
![图片](https://user-images.githubusercontent.com/95358444/144244938-107cf2cb-87e0-424d-a607-f6749d05071e.png)


As the game runs, the IK animation of the armature will be destroyed:
![图片](https://user-images.githubusercontent.com/95358444/144245049-3c72365c-f9c9-4dd0-a985-11e59a8e7105.png)
![图片](https://user-images.githubusercontent.com/95358444/144245096-348ce4b7-1a38-4857-925a-dd71188f3da6.png)
